tutorials/beginners-cookbook/index.md

```diff
@@ -293,71 +293,102 @@ var interGeo = geometry1.intersection(geometry2);
 var unGeo = geometry1.union(geometry2);
 ```
 
-#### Example: Geometry operations
+#### Example: Geometry operations [[Open In Code Editor](https://code.earthengine.google.com/f50a903f93c04aec85de795d9210842b)]
 
 Let's run of some these operations over the the state of Connecticut, US using geometries of the public US counties feature collection available on Earth Engine:
 
+**1.** We begin by zooming to the region of interest and loading/creating the geometries of interest by extracting them from the corresponding features.
 ```javascript
 // Set map center over the state of CT.
 Map.setCenter(-72.6978, 41.6798, 8);
 // Load US county dataset.
-var countyData=ee.FeatureCollection('TIGER/2018/Counties');
+var countyData = ee.FeatureCollection('TIGER/2018/Counties');
 // Filter the counties that are in Connecticut (more on filters later).
-var countyConnect=countyData.filter(ee.Filter.eq('STATEFP', '09'));
+var countyConnect = countyData.filter(ee.Filter.eq('STATEFP', '09'));
 // Get the union of all the county geometries in Connecticut.
-var countyConnectDiss=countyConnect.union();
-// Add the layer to the map with a specified color and layer name.
-Map.addLayer(countyConnectDiss, {color: 'red'}, 'Chicago dissolved');
-// Find the rectangle that emcompasses the southernmost, westernmost,
-// easternmost, and northernmost points of the feature.
+var countyConnectDiss = countyConnect.union();
+// Create a circular area using the first county in the Connecticut  FeatureCollection.
+var circle = ee.Feature(countyConnect.first()).geometry().centroid().buffer(50000);
```

Comment:
Please add non-zero error margins to all these.

Reply:
(Good practice to do that throughout.)

Reply:
Added a max error of 100 m to all the geometry operations in the tutorial.